src/components/Slider/__test__/slider.spec.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { mount } from 'enzyme';
 import Slider from '../index';
-import Error from '../styled/error';
+import ErrorText from '../../../components/Input/styled/errorText';
```

Comment:
Here the import can be:
```import ErrorText from '../../Input/styled/errorText';```
because you are going out of the component folder and then in again.